### PR TITLE
Add dark mode/light theme - Vanilla.js 

### PR DESCRIPTION
**Pull request by 29deepanshutyagi**
issue #84

### CODE REVIEW

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]
Thanks for the contribution @29deepanshutyagi , but I can't approve and merge. You might need to rebase. It seems like you're adding changes to GitHub Actions and Code Owner files. @finn-tbd just updated these, and we don't want those changes to revert. There's about 57 files changed here that I think will revert our repository.

Please remove all changes that are irrelevant to adding dark mode and light mode.

File: .github/workflows/vc-to-dwn-server-container-image.yml [hunk 1]
```diff
@@ -24,10 +24,10 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
 
       - name: Log in to the container registry
-        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446 # v3.2.0
```
Comment (blackgirlbytes) [inline .github/workflows/vc-to-dwn-server-container-image.yml:30]: There shouldn't be changes here either!

**Comment by blackgirlbytes** [comment]
If this is too much to rebase, I might suggest closing this PR, pulling the latest changes from our repo, creating a new branch with your changes, and opening a new PR.

**Review by blackgirlbytes — CHANGES_REQUESTED** [review]
My last comment is...I don't see the support for dark mode / light mode reflected in your PR unfortunately. I don't see a button to manually test it. 

Screenshot 2024-10-15 at 11 34 51 AM

And then please remember for each of your PRs to include a test, please and thank you. So we need a written test to make sure that light and dark mode are working.

**Comment by 29deepanshutyagi** [comment]
ok

**Pull request closed without merging**